gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/language/translator/GremlinTranslatorTest.java

```diff
@@ -151,6 +151,15 @@ public static Collection<Object[]> data() {
                             null,
                             "g.with_(\"x\")",
                             "g.with_('x')"},
+                    {"g.inject(UUID(\"f47ac10b-58cc-4372-a567-0e02b2c3d479\"))",
```

Comment:
It would be good to add equivalent translation tests for something like `"g.inject(UUID())"` to ensure it properly translates to a new random UUID in each of the GLVs (something like `"g.Inject<object>(Guid.NewGuid())"` in .NET)